src/rulesets/rest/2022-05-25/json-api-rules/resource-object-rules.ts

```diff
@@ -142,6 +142,24 @@ const matchRelationshipModificationRequestData = {
   },
 };
 
+// Relationship POST, PATCH, and DELETE requests must have
+// a request body with resource objects for the relationships
+// to be added/patched/deleted.
```

Comment:
nit: this comment isn't all that helpful when it is exactly the same as the one above